### PR TITLE
Update prerequisites.md

### PR DESCRIPTION
**Pull request by rathorevaibhav**
Targets #995 

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -122,3 +122,45 @@ php artisan migrate
...
 
        - Restart XAMPP. Next, open this url in your browser: http://portal.test
+    
+    2. For MAMP(Mac OS)
+       - Go to `etc/hosts` file or edit this in the terminal use the following command
+            ```
+            sudo nano /etc/hosts
+            ```
+            Add
+
```
Comment (P4NK4J) [inline docs/installation.md:132]: @kickb71 I think the statement seems missing, I was able to interpret that we need to add something in the hosts file, but exactly what to add, I could not figure out. Let me know your thoughts.

**Review by P4NK4J — COMMENTED** [review]
Left a comment, please check

File: docs/installation.md (hunk 1 above)
Comment (kickb71) [reply docs/installation.md:132]: I like to add  some comments there to 
May be the description of the host why is it there or may be what's the use of it 
May be the another way to access these files without the terminal file location

File: docs/installation.md [hunk 2]
```diff
@@ -122,3 +122,48 @@ php artisan migrate
...
+            And search for ‘vhosts‘ and uncomment line like this
+            ```
+            # Virtual hosts
+            #Include /private/etc/apache2/extra/httpd-vhosts.conf
+            ```
+            Change this to this
+            ```
+                # Virtual hosts
+                Include /private/etc/apache2/extra/httpd-vhosts.conf
+            ```
```
Comment (GauravGusain98) [inline docs/installation.md:149]: Please correct the indentation

Comment (GauravGusain98) [reply docs/installation.md:149]: Can you also correct the indentation on line 76 to 78

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Looks good, need some minor changes.

File: docs/installation.md (hunk 2 above)
Comment (gautam071) [reply docs/installation.md:149]: Yea Sure

File: Modules/HR/Http/Controllers/Recruitment/ReportsController.php [hunk 3]
```diff
@@ -50,8 +50,8 @@ public function searchBydate(Request $req)
             \DB::raw('MONTHNAME(created_at) as month_created_at'),
             \DB::raw('DATE(created_at) as date_created_at')
         )
-            ->where('created_at', '>=', $req->from)
-            ->where('created_at', '<=', $req->to)
+            ->where('created_at', '>=', $req->report_start_date)
+            ->where('created_at', '<=', $req->report_end_date)
```
Comment (GauravGusain98) [inline Modules/HR/Http/Controllers/Recruitment/ReportsController.php:54]: remove these changes from this PR.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please remove extra changes from this PR

File: Modules/HR/Http/Controllers/Recruitment/ReportsController.php (hunk 3 above)
Comment (gautam071) [reply Modules/HR/Http/Controllers/Recruitment/ReportsController.php]: Removed

**Review by GauravGusain98 — APPROVED** [review]
Looks good

**Pull request merged**